benchmark_runner/common/template_operations/templates/bootstorm/bootstorm_data_template.yaml

```diff
@@ -0,0 +1,22 @@
+metadata:
+  name: bootstorm
+template_data:
+  shared:
+    pin_node: {{ pin_node1 }}
+    odf_pvc: {{ odf_pvc }}
+    uuid: {{ uuid }}
+    fedora36_container_disk: quay.io/ebattat/fedora36-container-disk:latest
```

Comment:
Maybe `fedora_container_disk` so you don't have to rewrite a lot of code when you upgrade?

Reply:
Sorry, I meant this:
```
    fedora_container_disk: quay.io/ebattat/fedora36-container-disk:latest
```

So that if you later move to Fedora 37, you don't need to rewrite code everywhere.